src/puppetlabs/ring_middleware/core.clj

```diff
@@ -209,7 +295,7 @@
                                                :message msg}})
                         :plain (plain-response code msg))))]
      (fn [request]
-       (try (handler request)
+       (sling/try+ (handler request)
```

Comment:
What's the reason for this change?  I don't know exactly what `try+` does, but I think it just allows you to do fancier stuff in `catch`, which isn't happening here.